src/vehicle_use.cpp

```diff
@@ -983,6 +983,7 @@ bool vehicle::fold_up()
         bicycle.set_var( "volume", total_folded_volume() / units::legacy_volume_factor );
         bicycle.set_var( "name", string_format( _( "folded %s" ), name ) );
         bicycle.set_var( "vehicle_name", name );
+        bicycle.set_var( "tracking", tracking_on ? 1 : 0 );
```

Comment:
Unfortunately, this does not work for `folding bicycle`, as spawned as an item.

Just finished a fix for #59334 when I saw this PR. I `set_var` outside `if( can_be_folded )` condition, and it works there. (Honestly, it is not really clear to me what `can_be_folded` means here.) See the diff here: https://github.com/CleverRaven/Cataclysm-DDA/compare/master...mlange-42:Cataclysm-DDA:track-folded-vehicle?expand=1

Reply:
Seems `can_be_folded` is the "new" folding vehicles, `convertible` is the "old" hacky folding bicycle which has like 4 bug reports assigned to it and needs another place to handle that

Adjusted to handle both types according to your snippet

I tried to do the coauthoring thing, not sure it actually worked but it shows your icon next to the new commit now